packages/shared/lib/core/profile/actions/active-profile/logout.ts

```diff
@@ -1,6 +1,7 @@
 import { closePopup } from '@auxiliary/popup'
 import { resetSelectedAccount } from '@core/account'
-import { clearPollGovernanceDataInterval } from '@contexts/governance'
+import { clearPollGovernanceDataInterval } from '@contexts/governance/actions'
+import { resetLatestGovernanceTransactionIds } from '@contexts/governance/stores'
 import { isPollingLedgerDeviceStatus, stopPollingLedgerNanoStatus } from '@core/ledger'
 import { clearPollMarketPrices } from '@core/market/actions'
 import { clearPollNetworkInterval } from '@core/network'
```

Comment:
Might as well adjust the import specificity since we're here

Reply:
huh?